api/app/signals/apps/feedback/managers.py

```diff
@@ -9,4 +9,4 @@
 
 class FeedbackManager(models.Manager):
     def request_feedback(self, signal):
-        return self.create(**{'_signal': signal})
+        return self.create(**{'_signal': signal, 'allows_contact': True})
```

Comment:
Why did you add this? It is not needed, please revert back to how it originally was.